### ISSUE TITLE
for responsive is not working

### ISSUE THREAD

**Issue by Satosh-J** [femioladeji/react-slideshow#185]
```
      responsive={[
        {
          breakpoint: 1000,
          settings: {
            slidesToScroll: 3,
            slidesToShow: 5
          }
        },
        {
          breakpoint: 1200,
          settings: {
            slidesToScroll: 3,
            slidesToShow: 5
          }
        },
        {
          breakpoint: 1400,
          settings: {
            slidesToScroll: 3,
            slidesToShow: 5
          }
        },
        {
          breakpoint: 1600,
          settings: {
            slidesToScroll: 3,
            slidesToShow: 5
          }
        },
      ]}
```

I used this to change number of slides and Scroll count, but it always displays only on slide.
Is this working now?

**Comment by femioladeji** [comment]
@Satosh-J we'll look into this and get back to you

**Comment by femioladeji** [comment]
@Satosh-J thanks for raising this issue. There was indeed a bug. It has been fixed in the newly released version 4.0.3

You can check out the documentation for this prop here https://react-slideshow-image.netlify.app/?path=/story/examples-responsive--page

Let me know if it helps with what you were trying to accomplish
